api/metric/sdkhelpers.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metric
```

Comment:
I like how this turned out!

I think it would be nice if we moved these helpers into a sub-package. It's somewhat problematic, since this code uses un-exported types. Perhaps we could refactor this package to have an `./internal` package, a `./sdksupport` package (with this stuff), and a `./testsupport` package with the mock Meter. 

OK to postpone this to another PR.

Reply:
I find it hard to do the splitting, I'd postpone it.